source/colors.js

```diff
@@ -15,255 +15,354 @@ module.exports = {
+        { tone: '40', value: '#fe6100' },
+        { tone: '50', value: '#c45433' },
+        { tone: '60', value: '#993a1d' },
+        { tone: '70', value: '#782f1c' },
+        { tone: '80', value: '#56251a' },
+        { tone: '90', value: '#3a201b' }
       ]
     },
     {
-      name: 'yellow',
+      name: 'red',
       values: [
-        { tone: 'core', value: '#fdd600' },
-        { tone: '10', value: '#fde876' },
-        { tone: '20', value: '#fdd600' },
-        { tone: '30', value: '#efc100' },
-        { tone: '40', value: '#be9b00' },
-        { tone: '50', value: '#8c7300' },
-        { tone: '60', value: '#735f00' },
-        { tone: '70', value: '#574a00' },
-        { tone: '80', value: '#3c3200' },
-        { tone: '90', value: '#281e00' },
-        { tone: '100', value: '#020100' }
+        { tone: 'core', value: '#e62325' },
+        { tone: '1', value: '#f7e6e6' },
+        { tone: '10', value: '#fccec7' },
+        { tone: '20', value: '#ffaa9d' },
+        { tone: '30', value: '#ff806c' },
+        { tone: '40', value: '#ff5c49' },
+        { tone: '50', value: '#e62325' },
+        { tone: '60', value: '#aa231f' },
+        { tone: '70', value: '#83231e' },
+        { tone: '80', value: '#5c1f1b' },
+        { tone: '90', value: '#3e1d1b' }
       ]
     },
     {
-      name: 'aqua',
+      name: 'magenta',
       values: [
-        { tone: 'core', value: '#1e818c' },
-        { tone: '10', value: '#b6eff2' },
-        { tone: '20', value: '#7de1eb' },
-        { tone: '30', value: '#4cc3d2' },
-        { tone: '40', value: '#2ea4b2' },
-        { tone: '50', value: '#1e818c' },
-        { tone: '60', value: '#186a73' },
-        { tone: '70', value: '#0e5158' },
-        { tone: '80', value: '#0c373c' },
-        { tone: '90', value: '#07272b' },
-        { tone: '100', value: '#010203' }
+        { tone: 'core', value: '#ff509e' },
+        { tone: '1', value: '#f5e7eb' },
+        { tone: '10', value: '#f5cedb' },
+        { tone: '20', value: '#f7aac3' },
+        { tone: '30', value: '#f87eac' },
+        { tone: '40', value: '#ff509e' },
+        { tone: '50', value: '#dc267f' },
+        { tone: '60', value: '#a91560' },
+        { tone: '70', value: '#831b4c' },
+        { tone: '80', value: '#5d1a38' },
+        { tone: '90', value: '#401a29' }
+      ]
+    },
+    {
+      name: 'purple',
+      values: [
+        { tone: 'core', value: '#c22dd5' },
+        { tone: '1', value: '#f7e4fb' },
+        { tone: '10', value: '#efcef3' },
+        { tone: '20', value: '#e4adea' },
+        { tone: '30', value: '#d68adf' },
+        { tone: '40', value: '#cb71d7' },
+        { tone: '50', value: '#c22dd5' },
+        { tone: '60', value: '#9320a2' },
+        { tone: '70', value: '#71237c' },
+        { tone: '80', value: '#501e58' },
+        { tone: '90', value: '#3b1a40' }
+      ]
+    },
+    {
+      name: 'violet',
+      values: [
+        { tone: 'core', value: '#7d24c9' },
```

Comment:
Perhaps I'm misunderstanding then: are the `core` values not always equal to one of the `tone`s?